cuebot/src/main/resources/public/dtd/cjsl-1.8.dtd

```diff
@@ -15,11 +15,12 @@
 <!-- ================================================================= -->
```

Comment:
Should we make the same change to the other spec versions in this `dtd/` directory?

The last change here, #414, made the change to all versions. I assume the older versions are only in use at SPI so maybe @gregdenton has more info on the decision here.

Reply:
Yes I believe old versions are only in use at SPI, and it's mostly so old jobs can be rerun. I'm not sure this is really used though. I made the decision to update all because the addition was a breaking change. 
As a general rule I think we should aim to only update the "current" version.